src/Connections/SetupConnection.ts

```diff
@@ -41,6 +41,8 @@ export class SetupConnection extends CommandConnection {
                 roomId,
                 "",
                 "",
+                // TODO Consider storing room-specific config in state.
```

Comment:
Not sure what to do here, a SetupConnection has no state.

Reply:
True, but it may be useful for it to have state (like a custom command prefix).

The current situation with state is a bit wonky, though: the repo connection classes are stateful while SetupConnection is stateless, despite all of them being "command-controlled" while only the repo connections save their command prefix in state.  I couldn't think of a good way to share the state-setting code without it being inherited by SetupConnection as well (using mixins is an option, but is probably more trouble than it's worth), nor could I think of a way to separate out the code for handling command prefixes & saving state; so as it stands currently, SetupConnection holds an empty state object.